src/components/Form/DeleteDocumentForm.jsx

```diff
@@ -75,6 +72,14 @@ const DeleteDocumentForm = () => {
               docType={docType}
             />
             <br />
+            <TextField
```

Comment:
If the user have access to delete files in that container, they could see what files are already in that container. As such, it'll be better for users to select the file to delete rather than typing out the file name.

Reply:
To do this, we will need to implement fetch, which will be in a separate PR.